libs/designsystem/progress-circle/src/progress-circle-ring.component.ts

```diff
@@ -39,7 +30,18 @@ export class ProgressCircleRingComponent {
     return this._centerRadius * 2 * Math.PI;
   }
 
-  private calculateOffset(value: number): number {
-    return this._centerCircumference - this._centerCircumference * (value / 100);
+  get _progress(): number {
+    const valueWithinBounds = this.value < this.upperBound || this.value > 99;
+    const _value = valueWithinBounds ? this.value : this.upperBound;
+
+    return (_value * this._centerCircumference) / 100;
+  }
+
+  get _remainder(): number {
+    return this._centerCircumference - this._progress;
+  }
+
+  get _progressStrokeWidth(): number {
+    return this.value == 0 ? 0 : this.strokeWidth;
```

Comment:
Maybe add a comment as to why this is necessary - and/or implement with a guard to make the intent clearer:
```suggestion
    // Don't render stroke if value is 0, otherwise it will show as a dot:
    if (this.value === 0) return 0;
    return this.strokeWidth;
```